rpg.py

```diff
@@ -24,33 +24,42 @@ def __init__(self,
                  max_hp: int,
                  attack: int,
                  speed: int,
+                 leveling_rate: int,
+                 exp: int,
+                 level_exp: int,
                  team: int,
-                 level: int,
-                 exp: int,  # The exp value of the player
-                 target_exp: int):  # The target exp value required for the player to level up
+                 level: int):
         self.name = name
         self.hp = hp
         self.max_hp = max_hp
         self.attack = attack
         self.speed = speed
+        self.leveling_rate = leveling_rate
+        self.exp = exp
+        self.level_exp = level_exp
         self.team = team
         self.level = level
-        self.exp = exp
-        self.target_exp = target_exp
+        """:param max_hp: hp of an unharmed player
+        :param level: level of the player, 1-100 eventually
+        :param exp: value of the current exp a player has
+        :param target_exp: value of the exp needed for a player's level to increase"""
 
     def is_alive(self) -> bool:
         """Returns true if the player if they are alive"""
         return self.hp > 0
 
+    def is_burned(self) -> bool:
+        """check to see if the player has been burned"""
+        return self.is_burned is True
+
     def is_ally(self, other_player) -> bool:
         """returns true if the player is an ally"""
         return other_player.team == self.team
 
     def deal_damage(self, other_player):
         """Deals damage if the other player is alive"""
-        if other_player.is_alive():
+        if other_player.is_alive:
```

Comment:
You need `()` unless you made this a `property`. `is_alive` is still a method.

Why did you change this in the first place?